src/main/java/uk/gov/digital/ho/hocs/info/api/dto/ConfigurationDto.java

```diff
@@ -32,9 +32,8 @@ public static ConfigurationDto from(Configuration configuration) {
         String documentLabelsString = configuration.getDocumentLabels();
         List<String> documentLabels = StringUtils.isBlank(documentLabelsString) ? null : new ArrayList<>(Arrays.asList(documentLabelsString.split(",")));
 
-        String workstackColumnsString = configuration.getWorkstackColumns();
-        List<String> workstackColumns = StringUtils.isBlank(workstackColumnsString) ? null : new ArrayList<>(Arrays.asList(workstackColumnsString.split(",")));
 
+        List<WorkstackColumnDto> workstackColumns = configuration.getWorkstackColumns().stream().map(WorkstackColumnDto::from).collect(Collectors.toList());
         List<SearchFieldDto> searchFieldDtos = configuration.getSearchFields().stream().map(SearchFieldDto::from).collect(Collectors.toList());
 
         return new ConfigurationDto(
```

Comment:
I think there should be a test file for this method

Reply:
Done